aws/main.tf

```diff
@@ -0,0 +1,71 @@
+terraform {
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "~> 5.0"
+    }
+  }
+}
+
+locals {
+  yaml_secrets = yamldecode(file("../secrets.yaml"))
+  name_prefix = "/demo"
+}
+
+# Configure the AWS Provider
+provider "aws" {
+  region = "us-east-2"
+}
+
+// Github Pat Tokens
+resource "aws_ssm_parameter" "github_pat" {
+  name  = join("/", [local.name_prefix, "github/pat${count.index + 1}", ])
+  type  = "String"
+  count = 2
```

Comment:
Can we use `length(local.yaml_secrets.samples.github_pats)` instead of hard-coding the count? Same for other parameters with count

Reply:
oh sure! Good point